src/main/java/seedu/address/logic/commands/EditDentistCommand.java

```diff
@@ -99,24 +101,27 @@ public CommandResult execute(Model model) throws CommandException {
     }
 
     /**
-     * Creates and returns a {@code Person} with the details of {@code personToEdit}
-     * edited with {@code editPersonDescriptor}.
+     * Creates and returns a {@code Person} with the details of {@code personToEdit} edited with
```

Comment:
Should be "Creates and returns a {@code Dentist} with the details of {@code dentistToEdit} edited with"